src/schema-templates/recipe-description.block.php

```diff
@@ -0,0 +1,15 @@
+<?php
+/**
+ * Recipe description block schema template.
+ *
+ * @package Yoast\WP\SEO\Schema_Templates
+ */
+
+use Yoast\WP\SEO\Schema_Templates\Assets\Icons;
+
+// phpcs:disable WordPress.Security.EscapeOutput -- Reason: The Icons contains safe svg.
+?>
+{{block name="yoast/recipe-description" title="<?php esc_attr_e( 'Recipe description', 'wordpress-seo-premium' ); ?>" category="yoast-required-recipe-blocks" description="<?php esc_attr_e( 'The description of the recipe.', 'wordpress-seo-premium' ); ?>" icon="<?php echo Icons::heroicons_identification(); ?>" supports={"multiple": false} }}
+<div class="yoast-recipe-block__description {{class-name}}">
+	{{rich-text name="description" required=true tag="p" keepPlaceholderOnFocus=true placeholder="<?php esc_attr_e( 'Enter a recipe description...', 'wordpress-seo-premium' ); ?>"}}
```

Comment:
The placeholder text doesn't match the [designs](https://www.sketch.com/s/4c91f078-4831-45b9-8e95-ff171700ed27/a/zLe1Qr) (`...` should be absent).

Reply:
The text domain shouldn't be Premium